features/customer.feature

```diff
@@ -91,3 +91,26 @@ Scenario: Search for john
 #     Then I should see the message "Success"
 #     And I should see "Loki" in the results
 #     And I should not see "fido" in the results
+
+Scenario: Deleting a customer
+    When I visit the "home page"
+    And I set the "first_name" to "sally"
+    And I press the "Search" button
+    Then I should see the message "Success"
+    And I should see "sally" in the "first_name" field
+    When I press the "Delete" button
+    Then I should see the message "Success"
```

Comment:
the message here is not "Success" which causes behave to fail. In `rest_api.js` line 172, the message is "customer has been Deleted!"